AODN/AODN-WAVE-DM/DOT-WA-WAVE/waverider_library/wave_parameters_mapping.csv

```diff
@@ -0,0 +1,26 @@
+ORIGINAL_VARNAME,VARNAME,STANDARD_NAME,LONG_NAME,UNITS,ANCILLARY_VARIABLES,VALID_MIN,VALID_MAX,REFERENCE_DATUM,POSITIVE
+Total_Hs,WHTH,sea_surface_wave_significant_height,,m,,0,100,,
+Total_Tp,WPPE,sea_surface_wave_period_at_variance_spectral_density_maximum,,s,,0,50,,
+Total_Tm,WSMP,,sea_surface_wave_spectral_mean_period,s,,0,50,,
+Total_Tmax,WPPE,sea_surface_wave_period_at_variance_spectral_density_maximum,,s,,0,50,,
+Total_Ts,WPTH,sea_surface_wave_mean_period,,s,,0,50,,
```

Comment:
are Total_Tmax and Total_Ts awac or waverider variables? I didn't have them in my parameter list.

Reply:
They should be waverider mainly because it is under the waverider folder